doc/reference/trd-legal.md

```diff
@@ -124,16 +125,18 @@ copyright holder to relinquish the initial copyright.
 
 ### 4.1.1 Examples
 
-The common-case format is the top of a file with only license information:
+The common-case format is:
 
 ```rust
 // Licensed under the Apache License, Version 2.0 or the MIT License.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors <YYYY>.
+// Copyright <you/your company> <YYYY>.
 
 //! Module-level documentation...
 ```
 
```

Comment:
(I do not know how to use Github's suggestion feature for code containing code blocks, since the closing backticks of the code block terminate the suggestion early). I recommend changing this to say:

<pre>
The common-case format is:

```rust
// Licensed under the Apache License, Version 2.0 or the MIT License.
// SPDX-License-Identifier: Apache-2.0 OR MIT
// Copyright Tock Contributors <YYYY>.
//! Module-level documentation...
```

placed at the top of the file.

If you wish to specifically call out the contribution by you or your company,
you may do so by adding another copyright line:

```rust
// Licensed under the Apache License, Version 2.0 or the MIT License.
// SPDX-License-Identifier: Apache-2.0 OR MIT
// Copyright Tock Contributors <YYYY>.
// Copyright <you/your company> <YYYY>
//! Module-level documentation...
```

<post>

The justification is that I do not think that that most contributors will care about specifying copyright for themselves, and I do not think we should encourage it by default since doing so lengthens the license header and makes it more likely that it will fall out of date

Reply:
Gah, bad timing. I'll send another PR (or update my git repo to push directly to `tock/tock` and push a new commit).

Reply:
I pushed a fix in https://github.com/tock/tock/pull/3318/commits/762ac743f034410c8470a63c37f6f2847ccecb03

Reply:
Thanks, sorry about that